pom.xml

```diff
@@ -46,7 +46,7 @@
         <junit.version>4.12</junit.version>
         <hamcrest.version>1.3</hamcrest.version>
 
-        <hazelcast.version>4.0</hazelcast.version>
+        <hazelcast.version>5.3.6</hazelcast.version>
```

Comment:
is this necessary? if not, we keep it 4.0 for compatibility

Reply:
Fixed at https://github.com/hazelcast/hazelcast-remote-controller/pull/72/commits/de4a7c8ea2af169822d9960fbf7908993588e84f